### DISCUSSION TITLE
How to change the timeout for supabase webhooks?

### DISCUSSION THREAD

**Discussion by FilipeMarch** [supabase/supabase#15467] [Questions]
Currently the timeout is 1000ms, which is very small. How can I make the timeout for webhooks bigger?

**Comment by anarkrypto** [comment]
They could add an option to change the timeout.
Currently, when we create a HTTP Webhook through the Supabase web interface, it performs a POST to the following url:

`https://api.supabase.io/platform/pg-meta/[YOUR_DB_ID]/triggers`

Passing the webhook arguments in the payload, including the timeout (default 1000ms):

```json
{
    "events": [
        "INSERT"
    ],
   ...,
    "function_args": [
        "https://your-webhook-url/api/example",
        "POST",
        {"Content-type":"application/json"},
        "{}",
        "1000"
    ]
}
```

**Comment by FilipeMarch** [comment]
It seems now supabase supports timeout for webhooks. That's great.
![image](https://github.com/supabase/supabase/assets/23220309/3c712774-a772-4d7b-94c2-3963410e5d75)
Closing
